_posts/releases/2017-10-02-release-2.4.md

```diff
@@ -0,0 +1,54 @@
+---
+title:  "Singularity 2.4 Release"
+category: releases
+permalink: "release-2-4"
+version: "2.4"
+---
+
+Singularity 2.4 is relesed, and here are the important notes for you to know about!
+
+<img src="/assets/img/posts/releases/singularity-egg-easy.png">
+
+
+**Singularity Build**
+Building an image is now more intuitive with the introduction of our `build` command. You don't need to worry about using `create`, or `bootstrap`, you just build. It looks like this:
+
+```
+sudo singularity build container.simg Singularity
+```
+
+If you are tired of needing to estimate your image size, that issue goes away with build. The default format for a Singularity image base is now a smaller, compressed file format called **SquashFS**.  This format is also a much better fit for reproducible containers, as it is immutable. For more details, see our [build](/docs-build-container) documentation.
+
+**Build Bases**
+We originally supported just using an existing distribution package or docker as a base for building images. With 2.4, we have extended that set to include another Singularity image (`localimage`), a Singularity Hub or Registry image (`shub`) or the host operating system (`self`).
+
+**Instance Support**
+Singularity now has support for instances, which means services! Images instances can be started, stopped, and listed. Along with instances comes **Network Namespace Isolation**. We are excited to see the kinds of use cases that this will afford for our users. 
+
+**Singularity Registry**
+If you want to host your own Registry, we listened and you are in luck! Singularity <a href="https://www.github.com/singularityhub/sregistry">Registry</a> is ready for your use, along with a <a href="https://singularityhub.github.io/containers">portal</a> for you to "register your registry" to make it easier to share your images. Singularity Hub, along with improved building and updated builders, will follow this release.
+
+**Internal Modalarity**
+This release includes support for the <a href="/docs-apps">Standard Container Integration Format (SCI-F)</a> that makes it easy for you to produce single, internally modular and programmatically discoverage containers. More information is provided on it's <a href="https://containers-ftw.github.io/SCI-F/">main site</a>.
```

Comment:
Modularity

Reply:
actually let me just look at this locally - it's annoying to the PR person to have to make these little adjustments!

Reply:
Either way.  

Also, note that I don't think this means the docs are finished.  I still plan to continue polishing them.  But we just need to get something out so the users have a guide regardless of whether it is perfect.